core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -11,21 +11,7 @@
       <input id="topicName" type="text" class="form-control protractor-test-topic-name-field"
              ng-disabled="!topicRights.canEditTopic()"
              ng-model="editableName" ng-blur="updateTopicName(editableName)"
-             placeholder="Enter a name for the topic." maxlength="35" ng-trim="false">
-      <span class="form-text" style="font-size: smaller; color: red;" ng-if="editableName.length === 35">
```

Comment:
I removed these info messages for all modals since the input field anyway hard limits the length to whatever is specified, so these error messages don't serve any particular use and if the entered text is at the limit, these messages persist which wouldn't look good.

Reply:
Hm. I actually found these messages quite useful, to be honest, since they told me exactly what I was doing wrong and gave me info I needed to know. Perhaps reconsider?

Reply:
But these error messages were shown at the limit, right? So, if a title matches the limit, then this message would be shown always. Wouldn't that be bad?

Reply:
Ah I see. Yes I agree that it doesn't make sense to show them at the limit. 

But what happens if a creator types above the limit? Probably the message should show and the corresponding save button for the field should be disabled? Or we prevent the character from being typed and show a warning until a further change is made to the input field?

I think maybe you can use the existing Angular validation functionality too (ngMessages I think).

Reply:
With the 'maxlength' attribute, typing above the limit is prevented in the input field itself, so the user cannot type above the limit at all. Maybe, I can add a greyed out subtitle mentioning the limit then, to make it clear, is that fine?

Reply:
Yes, seems reasonable to me. Thanks!

Reply:
Done